level_2/1024.py

```diff
@@ -44,7 +44,7 @@ def cast_vote(url, cookie, payload):
 
 get_initial_data(endpoint, hodor_cookie, hodor_payload)
```

Comment:
Use `if __name__ == "__main__":` to make your code executable only when call it directly and not when import the file